### PR TITLE
Added solar to frontier

### PR DESCRIPTION
**Pull request by bjaanes**

### CODE REVIEW

File: packages/web/config/chain-infos.ts [hunk 1]
```diff
@@ -491,6 +491,16 @@ const chainInfos = (
           coinDecimals: 6,
           coinImageUrl: "/tokens/bjuno.png",
         },
+        {
+          type: "cw20",
+          contractAddress:
+            "juno159q8t5g02744lxq8lfmcn6f78qqulq9wn3y9w7lxjgkz4e0a6kvsfvapse",
+          coinDenom: "Solar",
```
Comment (JeremyParish69) [inline packages/web/config/chain-infos.ts:498]: This should be all caps, like a ticker symbol. Perhaps 'SOLAR' ?

**Comment by JeremyParish69** [comment]
can you please register this to the Cosmos Chain Registry? https://github.com/cosmos/chain-registry

**Comment by bjaanes** [comment]
@JeremyParish69 
Denom fixed. Created PR to chain-registry: https://github.com/cosmos/chain-registry/pull/693

**Comment by bjaanes** [comment]
@JeremyParish69 it has been merged in the chain-registry (by you I guess :))

**Review by JeremyParish69 — APPROVED** [review]
LGTM

**Pull request merged**